matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/timeline/TimelineEvent.kt

```diff
@@ -65,7 +65,7 @@ data class TimelineEvent(
 
     init {
         if (BuildConfig.DEBUG) {
-            assert(eventId == root.eventId)
+//            assert(eventId == root.eventId)
```

Comment:
`this.eventId` should always have the same value than `this.root.eventId`. In which case this was not true on your side?

Reply:
I wanted to use some code from LocalEventFactory to avoid the code duplication. In ```createEvent```, a new eventId is created for the event. 
In my case i don't know if i should use the past id even if i updated the relatedTo part ? or should i create a new id ? 🤔 
Knowing i never update the DB, i juste update the object before print it.

Reply:
@ganfra told me that i should keep the id.